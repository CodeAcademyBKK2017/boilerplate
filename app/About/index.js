import App from './app';
import React from 'react';
import {TabNavigator} from 'react-navigation';

const MyApp = TabNavigator({
  App: {
    screen: App
  }
}, {
  tabBarPosition: 'bottom',
  animationEnabled: true,
  tabBarOptions: {
    activeTintColor: '#e91e63'
  }
});
export default MyApp;
  