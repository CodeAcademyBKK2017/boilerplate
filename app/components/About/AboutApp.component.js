import React, {Component} from 'react';
// import styles from './AboutApp.style';
import {
  Text, View
} from 'react-native';

export default class AboutApp extends Component {
  render () {
    return (
      <View>
        <Text>
          About App
        </Text>
      </View>
    );
  }
}
