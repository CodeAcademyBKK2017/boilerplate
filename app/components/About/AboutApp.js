import React from 'react';
import {
  Text
} from 'react-native';
  
export default class MyNotificationsScreen extends React.Component {
    static navigationOptions = {
      tabBarLabel: 'APP'
    };
    render () {
      return (
        <Text>APP</Text>
        // <Button
        //   onPress={() => this.props.navigation.goBack()}
        //   title='Go back home'
        // />
      );
    }
}
