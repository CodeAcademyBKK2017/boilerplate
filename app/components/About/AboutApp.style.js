import {
  StyleSheet
} from 'react-native';

const styles = StyleSheet.create({
  container: {
    width: '100%',
    alignItems: 'stretch'
  },
  aboutButtonTouch: {
    backgroundColor: 'white',
    borderColor: 'black',
    borderTopWidth: 0.5
  },
  aboutButtonContent: {
    paddingVertical: 10,
    paddingHorizontal: 10,
    textAlign: 'center'
  }
});

export default styles;
