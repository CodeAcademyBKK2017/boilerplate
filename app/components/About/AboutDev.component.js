import React, {Component} from 'react';
// import styles from './AboutDev.style';
import {
  Text, View
} from 'react-native';

export default class AboutDev extends Component {
  render () {
    return (
      <View>
        <Text>
          About Dev
        </Text>
      </View>
    );
  }
}
