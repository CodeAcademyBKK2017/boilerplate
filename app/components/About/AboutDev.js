import React from 'react';
import {
  Text
} from 'react-native';
  
export default class MyNotificationsScreen extends React.Component {
    static navigationOptions = {
      tabBarLabel: 'DEV'
    };
    render () {
      return (
        <Text>DEV</Text>
        // <Button
        //   onPress={() => this.props.navigation.goBack()}
        //   title='Go back home'
        // />
      );
    }
}
