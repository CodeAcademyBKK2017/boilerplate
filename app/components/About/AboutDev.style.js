import {StyleSheet} from 'react-native';

export default StyleSheet.create({
  container: {
    flex: 1,
    padding: 10,
    justifyContent: 'flex-start',
    alignItems: 'center',
    backgroundColor: '#F5FCFF'
  }
});