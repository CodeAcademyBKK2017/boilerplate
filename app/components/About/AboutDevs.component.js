import React, {Component} from 'react';
import styles from './AboutDevs.style';
import {Text, View} from 'react-native';

class AboutDevs extends Component {
  render () {
    return (
      <View  style={styles.container}>
        <Text>About the Devs </Text>
      </View>
    );
  }
}

AboutDevs.defaultProps = {
};
AboutDevs.propTypes = {
};

export default AboutDevs;
