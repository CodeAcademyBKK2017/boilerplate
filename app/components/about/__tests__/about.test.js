import 'react-native';
import About from '../about.component';
import React from 'react';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';
// import {shallow} from 'enzyme';

describe('About', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <About />
    );
    expect(tree).toBeDefined();
  });
  it('About: renders correctly', () => {
    const tree = renderer.create(<About />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  
});
