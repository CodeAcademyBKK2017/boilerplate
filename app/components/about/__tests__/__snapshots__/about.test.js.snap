// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About About: renders correctly 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Hello From About
  </Text>
</View>
`;
